Library/Homebrew/brew.rb

```diff
@@ -88,7 +88,7 @@ def require?(path)
   end
 
   if internal_cmd
-    Homebrew.send cmd.to_s.tr("-", "_").downcase
+    Homebrew::InstallCommand.send cmd.to_s.tr("-", "_").downcase
```

Comment:
Hi @MikeMcQuaid @bfontaine, to get that 'one class per command' working, I had to make this change. However, there is a bug here that ```brew <any_command>``` will fail if `any_command` is not under the class ```InstallCommands```, such as ```brew tests```. Any ideas on how to overcome this bug?